app/src/main/res/values/strings.xml

```diff
@@ -304,55 +304,55 @@
     <string name="shared_texts">Shared Texts</string>
     <string name="github" translatable="false">GitHub</string>
     <string name="join_us">Join Us</string>
-    <string name="visit_license_url">Visit License URL</string>
-    <string name="visit_web_page">Visit Web Page</string>
+    <string name="visit_license_url">Go to License URL</string>
+    <string name="visit_web_page">Visit Website</string>
     <string name="background_service">Background Service</string>
     <string name="empty_clients_list">Clients appear here</string>
     <string name="all_clients">All Clients</string>
     <string name="connect_to_wifi">Connect to Wi-Fi</string>
-    <string name="wifi_connection_notice">1- Open Wi-Fi settings
-        \n\n2- Wait until \'%s\' appears on the Wi-Fi list
-        \n\n3- Pick it (%s) and paste the password (%s)
-        \n\n4- Return here after successful connection
+    <string name="wifi_connection_notice">1- Open Wi-Fi settings.
+        \n\n2- Wait until \"%s\" appears on the Wi-Fi list.
+        \n\n3- Pick it (%s) and paste the password (%s).
+        \n\n4- Return here after connecting.
     </string>
-    <string name="copy_password_and_open_settings"><![CDATA[Copy Password & Open Settings]]></string>
+    <string name="copy_password_and_open_settings"><![CDATA[Copy Password and Open Settings]]></string>
     <string name="wifi_password">Wi-Fi Password</string>
     <string name="notifications_instructive">Instructive Notifications</string>
     <string name="connect_to_wifi_notice">Connect to \"%s\"</string>
     <string name="enter_wifi_password_notice">Paste the password \"%s\"</string>
     <string name="not_connected_yet_notice">Not connected yet</string>
-    <string name="location_service_disabled_notice">Location Service is disabled</string>
-    <string name="wifi_disabled_notice">Wi-Fi is disabled</string>
+    <string name="location_service_disabled_notice">Location service off</string>
+    <string name="wifi_disabled_notice">Wi-Fi off</string>
     <string name="write_system_settings_permission_required_notice">Needs permission to modify system settings</string>
-    <string name="connecting_to_client">Connecting to Client</string>
+    <string name="connecting_to_client">Connecting to Client…</string>
```

Comment:
Can we go with "Connect to Client" here since this is also a title? I might add another string if needed.

Reply:
I don't like the word "client", but OK.

Reply:
```suggestion
    <string name="connecting_to_client">Connect to Client</string>
```